### DISCUSSION TITLE
Binding SONOFF ZBMINIL2 to Hue Dimmer Switch only works when turning on, but not off?!?!

### DISCUSSION THREAD

**Discussion by wexman** [Koenkk/zigbee2mqtt#18329] [Devices]
Hi,
please forgive me if I'm doing something dumb, I'm quite new to this...
I have made a binding for the onOff cluster from the switch to the sonoff. But it only works ONE WAY, i. e. if the sonoff is off, pressing the on button of the switch turns on the sonoff. But pressing the off button does not turn it off as I'd expected (nor does any of the other buttons). Turning it off via software and the little button on the sonoff itself *does* work, btw.

Am I missing something obvious?
Any other ideas?

Thanks in advance!

**Comment by bilak** [comment]
Hey, did you found a solution? I'm not even able to bind Hue dimmer to zbminil2.

**Comment by wexman** [comment]
No, I'm afraid not. I'm not even sure if this is a z2m issue at all...

Did you try to unbind the coordinator first?

**Comment by niek1992** [comment]
How do you have the ZBMini-L2 configured? By default it is configured as a Rocker switch, but by holding the small (and only) button for 3s you can change ZBMini-L2 to Momentary. Also could you explain what you're trying to achieve? Simple on/off toggle or would you like to dim when you toggle the ZBMini-L2 to "on" untill you have the desired brightness and then toggle to "off"?

**Comment by deviantintegral** [comment]
I have a ZBMINI and turning off via direct binding works fine. I'd be surprised if this regressed in the later revision.

Does it work if you bind to a group instead of directly to the ZBMINI?

I do have a similar problem with Sengled bulbs. I ended up writing an automation to handle turning them off.

One thing to consider; it seems like the Hue remotes may only allow group bindings when binding to a group. For example:

- Binding directly to two bulbs works.
- Binding to one group works.
- Binding to two groups works.
- Binding to one group and one device **does not work**.

**Reply by jojkos** [reply]
@deviantintegral would you have some example configuration or tutorial on how you managed to make it work? I am googling and trying everything, but i am unable to bind my ZBMINI or ZBMINIL2 to neither group nor light. Or rather i am able to set up the binding but it doesn't seem to be doing anything with the light when i toggle the switches.

**Reply by deviantintegral** [reply]
Here's the binding on the dimmer which is a https://www.zigbee2mqtt.io/devices/324131092621.html#philips-324131092621:

<img width="1618" alt="image" src="https://github.com/Koenkk/zigbee2mqtt/assets/255023/2811f9e9-93b2-4684-b877-5fedd6d61400">

Here it is added to the group `TV Lights`:

<img width="1615" alt="image" src="https://github.com/Koenkk/zigbee2mqtt/assets/255023/fa28a354-4ed9-4709-9ea3-58ef772d1df7">

Total aside, I just noticed this on the dimmer page.

> This method also works for Philips Hue Lightstrips, IKEA Trådfri, Sonoff ZBMINI, and Tuya light bulbs

Wow, this does make putting a zbmini in an inaccessible place much more reasonable!

**Reply by jojkos** [reply]
Thanks. I thought that the sonoff zbmini can be used as the controlling device, but it doesn't seem to work and it doesn't have onOff in the output clusters. I managed to at least bind ikea remote to a lightbulb, so now i finally know that something works. But what i need is a wall switch that could be bound to the lights and zbmini is probably not it :(

**Reply by deviantintegral** [reply]
I think Enbrighten switches should work for that. I have one bound to a Sengled power switch (so it turns on both the overhead lights and the china cabinet lights at the same time). I have another two Enbrighten switches bound to each other, essentially making two wired switches behave as a single switch.

https://www.zigbee2mqtt.io/devices/43076.html#enbrighten-43076
https://www.zigbee2mqtt.io/devices/43080.html#enbrighten-43080

**Reply by bilak** [reply]
> I have a ZBMINI and turning off via direct binding works fine. I'd be surprised if this regressed in the later revision.

Interesting, I've tried to hue dimmer switch to zbminil2 and OFF doesn't work for me. Turning it ON works without problem. What version of hue switch do you have and how did you bound it?

**Reply by deviantintegral** [reply]
It's a https://www.zigbee2mqtt.io/devices/324131092621.html#philips-324131092621 on fw `6.1.1.28573`. I have it bound as shown above in the "Family Room Dimmer" screenshot.

Are you binding to a group or a single device? Perhaps it doesn't like device bindings, I've never tried. Also, I've had exactly the same issue with some Sengled bulbs where only On works, and ended up just accepting it and creating an automation to handle the off case.

**Comment by artvolk** [comment]
Hi! 

I've had the same problem: SONOFF ZBMINI-L2 (firmware 1.0.14) can be turned on, but not off while bound to Hue Dimmer v1 or Hue Wall module. It works as expected (on/off) though bound to Ikea RODRET and with Ikea STYRBAR. For me it was direct binding to device (not group).

Hope this can help somebody.

**Reply by MikeTheBike38** [reply]
SONOFF ZBMINI-L2 + STYRBAR. Exactly the information i was looking for. Thank you!
